tests/libtransmission/file-test.cc

```diff
@@ -688,28 +674,26 @@ TEST_F(FileTest, pathResolve)
     tr_sys_path_remove(path1);
 
 #ifdef _WIN32
-
-    auto resolved = tr_sys_path_resolve("\\\\127.0.0.1\\NonExistent"sv, &error);
+    auto resolved = tr_sys_path_resolve(R"(\\127.0.0.1\NonExistent)", &error);
     EXPECT_EQ(""sv, resolved);
     EXPECT_TRUE(error);
     error = {};
 
-    resolved = tr_sys_path_resolve("\\\\127.0.0.1\\ADMIN$\\NonExistent"sv, &error);
+    resolved = tr_sys_path_resolve(R"(\\127.0.0.1\ADMIN$\NonExistent)", &error);
     EXPECT_EQ(""sv, resolved);
     EXPECT_TRUE(error);
     error = {};
 
-    for (auto const& input : { "\\\\127.0.0.1\\ADMIN$\\System32"sv,
-                               "\\\\127.0.0.1\\ADMIN$\\\\System32"sv,
-                               "\\\\127.0.0.1\\\\ADMIN$\\System32"sv,
-                               "\\\\127.0.0.1\\\\ADMIN$\\\\System32"sv,
-                               "\\\\127.0.0.1\\ADMIN$/System32"sv })
+    for (auto const& input : { R"(\\127.0.0.1\ADMIN$\System32)",
+                               R"(\\127.0.0.1\ADMIN$\\System32)",
+                               R"(\\127.0.0.1\\ADMIN$\System32)",
+                               R"(\\127.0.0.1\\ADMIN$\\System32)",
+                               R"(\\127.0.0.1\ADMIN$/System32)" })
     {
         resolved = tr_sys_path_resolve(input, &error);
         EXPECT_EQ("\\\\127.0.0.1\\ADMIN$\\System32"sv, resolved);
```

Comment:
Why not raw str here also?

Reply:
Not sure why this wasn't caught. Updated now.

Reply:
https://github.com/llvm/llvm-project/issues/133618